File: beginner/pagina13/3241/3241.ts
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8")
const lines = input.split("\n")
const n = Numbwer(lines.shift())

for(let i = 0; i < n; i++){
    let curr = lines.shift()

    if(curr === "P=NP"){
        console.log("skipped")
        continue
    }

    let [a, b] = curr.trim().split("+")
    a= Number(a)
    b= Number(b)

    console.log(a + b)
}

File: beginner/pagina13/3241/3241.rs
use std::io;

fn main() {
     let mut input = String::new();

     io::stdin().read_line(&mut input).unwrap();

     let n: usize = input.trim().parse::<usize>().unwrap();

     for _ in 0..n {
        input.clear();

        io::stdin().read_line(&mut input).unwrap();
        let curr: &str = input.trim();

        if curr = "P=NP" {
            println!("skipped");
            continue;
        }

        let nums: vec<u32> = curr.split("+").map(|x| x.parse::<u32>().unwrap()).collect();

        let a = nums[0];
        let b = nums[1];

        println!("{}", a + b);

     }
}